pkg/controller/install/install_controller.go

```diff
@@ -21,14 +29,44 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const (
+	// Namespace of Knative Serving
+	knativeServingNamespace = "knative-serving"
```

Comment:
Well, the namespace must match the namespace specified in uptream/released knative-serving-<vrsion>yaml, right?

Reply:
Not exactly. We filter the namespace from the upstream release to match whatever namespace you choose to install in, whether by using the `--namespace` or `--olm` operator flags.

Reply:
The actual namespace is reflected in the request namespace, correct?   If so, I can pick it up from request namespace.

Reply:
It might be, but it could also be overridden by the `--namespace` flag. The definitive place to find it is in the manifest. See that `checkForMinikube` fn for an example.